tern/classes/oci_image.py

```diff
@@ -0,0 +1,164 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2020 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import os
+import json
+import subprocess  # nosec
+from tern.utils import rootfs
+from tern.utils import general
+from tern.classes.image import Image
+from tern.utils.constants import manifest_file
+from tern.classes.image_layer import ImageLayer
+
+
+class OCIImage(Image):
+    '''A representation of an image created using OCI format
+    See image.py for super class's attributes
+    OCI Image specific attributes:
+        repotag: the repotag associated with this image
+        history: a list of commands used to create the filesystem layers
+        to_dict: return a dict representation of the object
+    '''
+
+    def __init__(self, repotag=None, image_id=None, image_name=None):
```

Comment:
I think we will have to use repotag here only. I will file an issue for that change to go through first.

Reply:
The second may be to see if we can use the registry API to get the image. I can look at this further.

Reply:
> I think we will have to use repotag here only. I will file an issue for that change to go through first.

@nishakm I have resolved this comment in the commit.